Makefile

```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
```

Comment:
@jmrodri is `K8S_VERSION` going to get updated to 1.20.2 in a follow-up? Or does that get updated in some other way downstream? It's purely cosmetic downstream, for those curious.